data_checker/README.md

```diff
@@ -0,0 +1,40 @@
+# Check of correct import of resource policies
+### We check ONLY if ANONYMOUS group has same READ access to ITEMS in Dspace5 and Dspace7 after success data import! We don't check other resource policies.
+
+## How to use it:
+1. Update `const.py`
+   - `user = "<ADMIN_NAME>"`
+   - `password = "<ADMIN_PASSWORD>"`
+
+   - `# http or https`
+   - `use_ssl = False`
+   - `host = "<YOUR_SERVER>" e.g., localhost`
+   - `# host = "dev-5.pc"`
+   - `fe_port = "<YOUR_FE_PORT>"`
+   - `# fe_port = ":4000"`
+   - `be_port = "<YOUR_BE_PORT>"`
+   - `# be_port = ":8080"`
+   - `be_location = "/server/"`
+   
+   - `authentication = False` - we test anonymous access
+
+   - `CLARIN_DSPACE_NAME = "clarin-dspace"`
+   - `CLARIN_DSPACE_HOST = "localhost"`
+   - `CLARIN_DSPACE_USER = "<USERNAME>"`
+   - `CLARIN_DSPACE_PASSWORD = "<PASSWORD>"`
+
+2. Be sure your project contains files:
+   **IMPORTANT:** If `data` or `temp-files` folders don't exist in the project, create them

```

Comment:
I noticed those folders are not in `.gitignore`. They should be. Either add them in this branch, or create an issue and add them later. But adding here should be ok, in my opinion.

Reply:
i don't understand.... should i create empty folders or what? this note is from general readme for data pump

Reply:
someone should add folders `data` and `temp-files` to `.gitignore`